cfme/tests/infrastructure/test_host.py

```diff
@@ -468,6 +468,43 @@ def test_compare_hosts_from_provider_allhosts(appliance, setup_provider_min_host
     assert compare_hosts_view.is_displayed
 
 
+@test_requirements.infra_hosts
+@pytest.mark.meta(blockers=[BZ(1747545, forced_streams=["5.10"])], automates=[1747545])
+@pytest.mark.meta(blockers=[BZ(1746214, forced_streams=["5.10"])], automates=[1746214])
```

Comment:
_Optional_ These can put into a single line

```suggestion
@pytest.mark.meta(blockers=[BZ(1746214, forced_streams=["5.10"]), BZ(1747545, forced_streams=["5.10"])], automates=[1746214, 1747545])
```

Reply:
**Required**

We only want one meta marker.

Reply:
I combined them.